package-lock.json

```diff
@@ -6,7 +6,7 @@
   "dependencies": {
     "@babel/code-frame": {
       "version": "7.10.4",
-      "resolved": "http://artifactory.inf.las1.saucelabs.net:80/artifactory/api/npm/all-npm/@babel/code-frame/-/code-frame-7.10.4.tgz",
+      "resolved": "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.10.4.tgz",
```

Comment:
Did you mean to completely remove artifactory or change to `https` from `http`?

If remove, is that going to start the old problem again of too much npmjs traffic?